apps/web/src/pages/integrations/IntegrationsListPage.tsx

```diff
@@ -63,10 +65,66 @@ const IntegrationsList = () => {
   const { integrations, loading: areIntegrationsLoading } = useIntegrations();
   const isLoading = areEnvironmentsLoading || areIntegrationsLoading;
   const hasIntegrations = integrations && integrations?.length > 0;
-  const data = useMemo<ITableIntegration[] | undefined>(
-    () => integrations?.map((el) => mapToTableIntegration(el, environments)),
-    [integrations, environments]
+
+  const {
+    data: { limit: emailLimit, count: emailCount },
+    loading: emailLimitLoading,
+  } = useIntegrationLimit(ChannelTypeEnum.EMAIL);
+  const {
+    data: { limit: smsLimit, count: smsCount },
+    loading: smsLimitLoading,
+  } = useIntegrationLimit(ChannelTypeEnum.SMS);
```

Comment:
these are the hooks to request the Novu providers limits for SMS and Email, that information is used to calculate if the integration is still active or not